scripts/e2etests/VSUtils.ps1

```diff
@@ -186,19 +178,20 @@ function ExecuteCommand {
 function GetVSIXInstallerPath {
     param(
         [Parameter(Mandatory = $true)]
-        [ValidateSet("15.0")]
+        [ValidateSet("16.0")]
         [string]$VSVersion
     )
 
     $VSIDEFolderPath = GetVSIDEFolderPath $VSVersion
     $VSIXInstallerPath = Join-Path $VSIDEFolderPath "$VSInstallerProcessName.exe"
 
-    return $VSIXInstallerPath
+    # TODO: This needs to be removed when https://developercommunity.visualstudio.com/content/problem/441998/vsixinstallerexe-not-working-in-vs2019-preview-20.html is fixed (it should be in Preview 4)
+    return "C:\Program Files (x86)\Microsoft Visual Studio\Installer\resources\app\ServiceHub\Services\Microsoft.VisualStudio.Setup.Service\VSIXInstaller.exe"
```

Comment:
I don't know if there's any reason why someone would want to run the e2e tests on their own machine, but further up in this same script we already have the variable `$ProgramFilesPath`, so maybe we should use that, just in case. Although as your comment says, the fix should be released pretty soon.

Reply:
The E2E tests page in our Engineering one note has "run locally" instructions. 

Basically if you want to run locally you are not expected to install the VSIX using the scripts. 
You are likely doing granular things.